tst/EnergyPlus/unit/SimAirServingZones.unit.cc

```diff
@@ -1033,6 +1033,142 @@ TEST_F(EnergyPlusFixture, InitAirLoops_2AirLoop2ADU)
+        "    AirLoopHVAC:ZoneMixer,   !- Component 1 Object Type",
+        "    Zone Return Air Mixer;   !- Component 1 Name",
+
+        "  AirLoopHVAC:ZoneMixer,",
+        "    Zone Return Air Mixer,   !- Name",
+        "    Demand Out Node,         !- Outlet Node Name",
+        "    Space1 Ret Node;         !- Inlet 1 Node Name",
+
+        "  Fan:VariableVolume,",
+        "    Supply Fan 1,            !- Name",
+        "    ,                        !- Availability Schedule Name",
+        "    0.7,                     !- Fan Total Efficiency",
+        "    600.0,                   !- Pressure Rise {Pa}",
+        "    autosize,                !- Maximum Flow Rate {m3/s}",
+        "    FixedFlowRate,           !- Fan Power Minimum Flow Rate Input Method",
+        "    ,                        !- Fan Power Minimum Flow Fraction",
+        "    0.0,                 !- Fan Power Minimum Air Flow Rate {m3/s}",
+        "    0.9,                     !- Motor Efficiency",
+        "    1.0,                     !- Motor In Airstream Fraction",
+        "    0.0015302446,            !- Fan Power Coefficient 1",
+        "    0.0052080574,            !- Fan Power Coefficient 2",
+        "    1.1086242,               !- Fan Power Coefficient 3",
+        "    -0.11635563,             !- Fan Power Coefficient 4",
+        "    0.000,                   !- Fan Power Coefficient 5",
+        "    VAV Sys 1 Inlet Node,    !- Air Inlet Node Name",
+        "    VAV Sys 1 Outlet Node;   !- Air Outlet Node Name",
+
+    });
+
+    ASSERT_TRUE(process_idf(idf_objects));
+    bool ErrorsFound = false;
+    HeatBalanceManager::GetZoneData(*state, ErrorsFound);
+    ASSERT_FALSE(ErrorsFound);
+    EXPECT_TRUE(compare_err_stream(""));
+    DataZoneEquipment::GetZoneEquipmentData(*state);
+    EXPECT_TRUE(compare_err_stream(""));
+    ASSERT_FALSE(ErrorsFound);
+    ZoneAirLoopEquipmentManager::GetZoneAirLoopEquipment(*state);
+    EXPECT_TRUE(compare_err_stream(""));
+    ASSERT_FALSE(ErrorsFound);
+    SingleDuct::GetSysInput(*state);
+    EXPECT_TRUE(compare_err_stream(""));
+    ASSERT_FALSE(ErrorsFound);
+    SplitterComponent::GetSplitterInput(*state);
+    EXPECT_TRUE(compare_err_stream(""));
+    SimAirServingZones::GetAirPathData(*state);
+    // Expect warnings about no controllers, clear err_stream
+    EXPECT_TRUE(has_err_output(true));
+    SimAirServingZones::InitAirLoops(*state, true);
+    // Expect warnings about 0 airflow
+    std::string error_msg = delimited_string({
+        "   ** Severe  ** SizeAirLoopBranches: AirLoopHVAC VAV SYS 1 has air flow less than 1.0000E-003 m3/s.",
+        "   **   ~~~   ** Primary air system volumetric flow rate = 0.0000 m3/s.",
+        "   **   ~~~   ** Check flow rate inputs for components in this air loop and,",
+        "   **   ~~~   ** if autosized, check Sizing:Zone and Sizing:System objects and related inputs.",
+    });
+    EXPECT_TRUE(compare_err_stream(error_msg));
+}
+
```

Comment:
new unit test created to make sure the correct error is given when AirLoop flow is 0.